kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -20,8 +20,11 @@ def __init__(self, model_part, params, file_header):
             "write_buffer_size" : -1
         }''')
         # write_buffer_size: -1 means we use the system default
+        # write_buffer_size:  0 means no buffering is done.
```

Comment:
this will throw an error, I think you should remove this comment
or say that it won't work

Reply:
I see it will throw an error but the error message is not explaining what the problem is. I will add explicit error message.